src/qtiCommonRenderer/renderers/interactions/ExtendedTextInteraction.js

```diff
@@ -58,11 +58,22 @@ const render = function render(interaction) {
 
         const placeholderText = interaction.attr('placeholderText');
 
+        const getItemLanguage = function getItemLanguage() {
+            let itemLang = window.document.documentElement.querySelector('.qti-item').getAttribute('lang');
+            let itemLocale = itemLang && itemLang.split('-')[0];
+            if (itemLocale) {
+                itemLang = window.document.documentElement.getAttribute('lang');
+                itemLocale = itemLocale && itemLocale.split('-')[0];
+            }
```

Comment:
**nitpick:** I don't see the point here, the `itemLang` is not used after it assignment. Then the `itemLocal` gets split once again...

Reply:
I've refactored this code, was complete wrong, my bad.